pages/samvera/developer_community/release_process.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Hyrax Testing and Release Process
-a-z: [Testing, Release Process, Managers, Product Owners]
+a-z: [Testing, Release Process, Managers, Product Owners, NURAX, NURAX-DEV]
```

Comment:
Having multiple pages with the same a-z terms is not very helpful. Only the term shows in the list multiple times, and the site user will not know where the link is taking them. Please re-evaluate the use of a-z terms to be more user-friendly.